doc/reference/backends.rst

```diff
@@ -2,18 +2,21 @@
 Backends and Configs
 ********************
 
-.. note:: Both NetworkX backend and config systems are experimental. They
+.. note:: Both NetworkX backend and config systems are experimental. Backends
```

Comment:
I think the original intent of the experimental status was more for backend developers than end users, meaning internal APIs used by backends may change, but the user-facing graph algos/functions are stable.  Perhaps this entire sentence is dropped?  If not dropped, can we reword it so users know the user-facing APIs (which is just the NX API itself) is stable?